pkg/pdutil/clock.go

```diff
@@ -91,30 +90,26 @@ func (c *clock) Run(ctx context.Context) {
 				c.mu.Lock()
 				c.mu.tsEventTime = oracle.GetTimeFromTS(oracle.ComposeTS(physical, 0))
 				c.mu.tsProcessingTime = time.Now()
-				c.mu.err = nil
 				c.mu.Unlock()
 				return nil
 			}, retry.WithBackoffBaseDelay(200), retry.WithMaxTries(10))
 			if err != nil {
-				log.Warn("get time from pd failed, will use local time as pd time")
-				c.mu.Lock()
-				now := time.Now()
-				c.mu.tsEventTime = now
```

Comment:
Why remove this?

Reply:
It's not necessary now. We can use `tsProcessingTime` to calculate approximate PD time when PD API does not response.